### PR TITLE
Mention only secret key in staking guide

### PR DESCRIPTION
**Pull request by ipopescu**
### Related links

Parent sprint card: https://github.com/casper-network/docs/issues/169

### Changes

- Updated instructions to download the secret key
- Updated screenshot for step 5 in section 3.2
- Removed algorithm recommendation and added a link to this section instead: https://casper.network/docs/dapp-dev-guide/keys
- Removed steps 6-9 and created a note instead
- Removed screenshot showing 3 files
- Removed the mention of Clarity
- Removed this “Please do this as early as possible!” since the signer now downloads the private key on account creation.

### Notes

The staking video will be updated as part of card [#192](https://github.com/casper-network/docs/issues/192).
I reviewed all other occurrences of key management in the docs.

### CODE REVIEW

File: source/docs/casper/workflow/staking.md [hunk 1]
```diff
@@ -59,40 +59,36 @@ If you are new or have logged out of the Signer, you can log in with these steps
...
 9.  You can also view websites and dApps to which your accounts are connected.
-10. One essential function is the ability to download your keys and store them in a secure location. If you lose access to the vault, you can create a new vault with the downloaded files. Please do this as early as possible!
+10. One essential function is the ability to download your keys and store them in a secure location. If you lose access to the vault, you can create a new vault with the downloaded files. 
 
 ### 3.2 Creating a New Account {#32-creating-a-new-account}
 
-If you are using the CasperLabs Signer for the first time, follow these steps to create an account and **download the account's keys**:
+If you are using the CasperLabs Signer for the first time, follow these steps to create an account and **download the account's secret key**:
 
 1.  Click **Create Account** in the Signer.
```
Comment (GloreenS) [inline source/docs/casper/workflow/staking.md:68]:
```suggestion
1.  Click **CREATE ACCOUNT** in the Signer.
```
The UI element is all caps.

File: source/docs/casper/workflow/staking.md [hunk 2]
```diff
@@ -59,40 +59,36 @@ If you are new or have logged out of the Signer, you can log in with these steps
...
 2.  Give your account a name. In this tutorial, we will use the name _My-CSPR_.
-3.  Select an Algorithm using the dropdown menu. Select _SECP256K1_ unless you have a good reason not to do so.
+3.  Select an Algorithm using the dropdown menu. If you need help, review the article on [Working with Cryptographic Keys](https://casper.network/docs/dapp-dev-guide/keys).
 4.  Click **Create** to save your account.
```
Comment (GloreenS) [inline source/docs/casper/workflow/staking.md:71]:
```suggestion
4.  Click **CREATE** to save your account.
```

File: source/docs/casper/workflow/staking.md [hunk 3]
```diff
@@ -59,40 +59,36 @@ If you are new or have logged out of the Signer, you can log in with these steps
...
-5.  **IMPORTANT: Download the key files for this account!** Click the hamburger icon and select the **Download Active Key** option.
+5.  Click **Download** and save the secret key for this account.
```
Comment (GloreenS) [inline source/docs/casper/workflow/staking.md:72]:
```suggestion
5.  Click **DOWNLOAD** and save the secret key for this account. 
```

**Review by GloreenS — APPROVED** [review]
I have left a few minor suggestions and this comment, rest of the changes look good. Thanks!
- In the following topic, the second sentence doesn't seem valid anymore, as there is no way to download the public key or the public key hex from the Signer.
**5.1 Transfer CSPR from an Exchange** 
If you need to transfer your CSPR tokens from an exchange, you will need your **public key** from the account page. ~~You can also find this key in the **public_key_hex** file, which you can download from the CasperLabs Signer.~~

**Comment by ipopescu** [comment]
Thanks, @GloreenS. Feedback addressed.

**Pull request merged**